### PR TITLE
[Page Header]Fix responsiveness issue for dashboards & visualization app headers

### PR DESCRIPTION
**Pull request by zhongnansu**
### Description

- [Page Header]Fix responsiveness issue for dashboards & visualization app header

https://github.com/user-attachments/assets/35f4eb5a-9ba4-4b00-8241-63a667d7fa31

## Changelog

- skip

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/data/public/ui/query_string_input/query_bar_top_row.tsx [hunk 1]
```diff
@@ -265,7 +265,7 @@ export default function QueryBarTopRow(props: QueryBarTopRowProps) {
 
     return (
       <NoDataPopover storage={storage} showNoDataPopover={props.indicateNoData}>
-        <EuiFlexGroup responsive={false} gutterSize="s" alignItems="flexStart">
+        <EuiFlexGroup responsive={false} gutterSize="none" justifyContent="flexStart">
```
Comment (virajsanghvi) [inline src/plugins/data/public/ui/query_string_input/query_bar_top_row.tsx:268]: this gutter removal didn't regress anything?

**Review by virajsanghvi — APPROVED** [review]
approving assuming comment isn't an issue and ci passes

File: src/plugins/data/public/ui/query_string_input/query_bar_top_row.tsx (hunk 1 above)
Comment (zhongnansu) [reply src/plugins/data/public/ui/query_string_input/query_bar_top_row.tsx:268]: this is to be consistent with existing date picker in discover. But let me check the UX mock up
image

Comment (zhongnansu) [reply src/plugins/data/public/ui/query_string_input/query_bar_top_row.tsx]: revert this back, also update discover header to use gutter size "s", based on UX mock, there should be spacing between refresh button and datepicker

**Comment by virajsanghvi** [comment]
Can you check why 
image
 gets misaligned?

**Comment by zhongnansu** [comment]
> Can you check why image gets misaligned on small screens per your video?

@virajsanghvi fixed the css issue and updated the PR

**Review by virajsanghvi — DISMISSED** [review]

**Comment by virajsanghvi** [comment]
note: May want to rerun ci for failed tasks

**Comment by zhongnansu** [comment]
> note: May want to rerun ci for failed tasks

@virajsanghvi Fixed. The stying issue of scss file. Do you have any recommendation for a good scss formatting tool? I remember researching for some, but not very good to use

**Review by zhyuanqi — APPROVED** [review]

**Review by Flyingliuhub — APPROVED** [review]

**Pull request merged**